### ISSUE TITLE
Check if pytrip can connect to trip

### ISSUE THREAD

**Issue by grzanka** [pytrip/pytrip#94]
Starting TRiP98 from pytrip might not work now, paramiko is not on a list of requirements.
Tests to connect to TRiP98 remotely are missing.

**Comment by nbassler** [comment]
Attempted to execute remotely: 

```
Traceback (most recent call last):
  File "examples/trip98_simple.py", line 83, in <module>
    my_trip.execute(my_plan)
  File "/home/bassler/Projects/pytrip/pytrip/tripexecuter/tripexecuter.py", line 104, in execute
    self.execute_simple()
  File "/home/bassler/Projects/pytrip/pytrip/tripexecuter/tripexecuter.py", line 127, in execute_simple
    self.run_trip()
  File "/home/bassler/Projects/pytrip/pytrip/tripexecuter/tripexecuter.py", line 574, in run_trip
    self.run_trip_remote()
  File "/home/bassler/Projects/pytrip/pytrip/tripexecuter/tripexecuter.py", line 580, in run_trip_remote
    self.compress_files()
  File "/home/bassler/Projects/pytrip/pytrip/tripexecuter/tripexecuter.py", line 680, in compress_files
    self.save_exec(os.path.join(self.working_path, self.folder_name, "plan.exec"))
  File "/home/bassler/Projects/pytrip/pytrip/tripexecuter/tripexecuter.py", line 697, in save_exec
    for projectile in self.projectiles:
AttributeError: 'TripExecuter' object has no attribute 'projectiles'
>>> 
```

**Comment by grzanka** [comment]
@nbassler  is it fixed ?

**Comment by nbassler** [comment]
Yes, was fixed with #130
